contracts/binding/src/lib.rs

```diff
@@ -2,47 +2,47 @@
 #[macro_use]
 mod convert;
 #[allow(clippy::all)]
-pub mod i_diamond;
+pub mod bottom_up_router_facet;
```

Comment:
Ah, look, here's a reason this changes: I think you are formatting the generated code. 

`build.rs` produces these modules in [alphabetical order](https://github.com/consensus-shipyard/ipc-monorepo/blob/c4e8e0eb5db12fe0aa13b207af375e76b7e1cbb6/contracts/binding/build.rs#L38-L73), and I specifically asked [rustfmt to ignore](https://github.com/consensus-shipyard/ipc-monorepo/blob/c4e8e0eb5db12fe0aa13b207af375e76b7e1cbb6/rustfmt.toml#L4) these files, yet it changes.